cfgov/apache/conf/httpd.conf

```diff
@@ -279,6 +276,9 @@ AddDefaultCharset UTF-8
 #EnableMMAP off
 EnableSendfile on
 
+# Enable mod_rewrite globally
```

Comment:
This might be unnecessary, but for consistency with the rest of this file, could we enclose this in an `<IfModule rewrite_module></IfModule>`?

Reply:
Fixed.